modules/webgl/src/classes/framebuffer.js

```diff
@@ -128,6 +129,12 @@ export default class Framebuffer extends Resource {
     }
   }
 
+  delete() {
+    for (const resource of this.resources) {
```

Comment:
Don't we already have references to these resources in `this.attachments`? 

I would prefer walking that object and deleting/clearing rather than adding a second set of references (that could get "out of sync" during maintenance of this class).

Reply:
`this.attachments` could be pointing to resources provided by the user of this class and not owned by it, so added a new list to track all resources created by this class.

Reply:
I guess that is a fair point. Since several people are working on resource managers, I am happy to defer to ongoing work in that area.

Reply:
I'd suggest renaming to something like `ownResources` then, to make it clear that it's resources managed by the framebuffer.